components/slider/index.css

```diff
@@ -455,17 +495,239 @@ governing permissions and limitations under the License.
+
+  &:hover {
+    border-color: var(--highcontrast-slider-handle-border-color-hover, var(--mod-slider-handle-border-color-hover, var(--spectrum-slider-handle-border-color-hover)));
+  }
+
+  &.is-focused {
+    border-color: var(--highcontrast-slider-handle-border-color-key-focus, var(--mod-slider-handle-border-color-key-focus, var(--spectrum-slider-handle-border-color-key-focus)));
+
+    &:before {
+      box-shadow: 0 0 0 var(--spectrum-focus-indicator-thickness) var(--highcontrast-slider-handle-focus-ring-color-key-focus, var(--mod-slider-handle-focus-ring-color-key-focus, var(--spectrum-slider-handle-focus-ring-color-key-focus)));
+    }
+  }
+
+  &:active,
+  &.is-dragged {
+    border-color: var(--highcontrast-slider-handle-border-color-down, var(--mod-slider-handle-border-color-down, var(--spectrum-slider-handle-border-color-down)));
+  }
+}
+
+
+.spectrum-Slider--ramp {
+  .spectrum-Slider-handle {
+    box-shadow: 0 0 0 var(--spectrum-slider-handle-gap) var(--highcontrast-slider-ramp-handle-border-color-active, var(--mod-sectrum-slider-ramp-handle-border-color-active, var(--spectrum-slider-ramp-handle-border-color-active)));
+
+    /* currently in tokens v5.0.0 the background-color for the handle are not yet defined */
+    background: var(--mod-slider-ramp-handle-background-color, var(--spectrum-slider-ramp-handle-background-color));
+
+    @media (forced-colors: active) {
+      /* forced-color-adjust required to ensure the "circle" around the handle is transparent */
+      forced-color-adjust: none;
+      box-shadow: 0 0 0 var(--spectrum-slider-handle-gap) ButtonFace;
+      border-color: ButtonText;
+      background-color: ButtonFace;
+    }
+  }
+}
+
+.spectrum-Slider-input {
+  background: transparent;
+}
+
+.spectrum-Slider-tick {
+  &:after {
+    background-color: var(--highcontrast-slider-tick-mark-color, var(--mod-slider-tick-mark-color, var(--spectrum-slider-tick-mark-color)));
+  }
+}
+
+.spectrum-Slider-handle {
+  &.is-dragged {
+    border-color: var(--highcontrast-slider-handle-border-color-down, var(--mod-slider-handle-border-color-down), var(--spectrum-slider-handle-border-color-down)));
+    background: var(--highcontrast-slider-handle-background-color, var(--mod-slider-handle-background-color), var(--spectrum-slider-handle-background-color)));
+  }
+}
+
+
+.spectrum-Slider--range {
+  .spectrum-Slider-track {
+    &:not(:first-of-type):not(:last-of-type) {
+      &:before {
+        background: var(--highcontrast-slider-track-fill-color, var(--mod-slider-track-fill-color), var(--spectrum-slider-track-fill-color)));
+      }
+    }
+  }
+}
+
+
+.spectrum-Slider {
+  &.is-disabled {
+    .spectrum-Slider-labelContainer {
+      color: var(--highcontrast-slider-label-text-color-disabled, var(--mod-slider-label-text-color-disabled, var(--spectrum-slider-label-text-color-disabled)));
+    }
+
```

Comment:
I am wondering if the tick labels should be added here like

```
    .spectrum-Slider-tickLabel {
      color: var(--highcontrast-slider-label-text-color-disabled, var(--mod-slider-label-text-color-disabled, var(--spectrum-slider-label-text-color-disabled)));
    }
```

To give the tick labels the same disabled color as the rest of the text. I've asked for feedback from design, we will see what they say.

Reply:
Hey @bernhard-adobe , Design has confirmed that the tick labels should be the same color as the other disabled labels. This could be added either here or on line 522.